hazelcast-sql/src/main/java/com/hazelcast/sql/impl/expression/ExpressionEvalContextImpl.java

```diff
@@ -44,21 +51,35 @@ public class ExpressionEvalContextImpl implements ExpressionEvalContext {
     public ExpressionEvalContextImpl(
             @Nonnull List<Object> arguments,
             @Nonnull InternalSerializationService serializationService,
-            @Nonnull NodeEngine nodeEngine) {
+            @Nonnull NodeEngine nodeEngine,
+            @Nonnull SqlSecurityContext ssc) {
```

Comment:
order of the arguments between constructors is inconsistent. Security-defining thing in one is first, in the other it is last.

Reply:
Actually, no. 
In first we have a meta-supplier, a dominant entity for EEC. Here `SqlSecurityContext` is just another argument.